spec/requests/api/v1/restaurants_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'rails_helper'
+
+RSpec.describe Api::V1::RestaurantsController, type: :request do
+  let(:headers) { { HTTP_ACCEPT: 'application/json' } }
+
+  describe 'GET /v1/restaurants' do
+    let!(:owner) do
+      create(:user, email: 'whatever@random_restaurant.com', role: 'owner')
+    end
+    let!(:owner_2) do
+      create(:user, email: 'whatever@random_restaurant2.com', role: 'owner')
+    end
+    let!(:restaurant) { create(:restaurant, user: owner) }
+    let!(:restaurant_2) { create(:restaurant, user: owner) }
+
+    it 'should return json with restaurants' do
+      get '/api/v1/restaurants'
+      json_response = JSON.parse(response.body)
```

Comment:
Could be extracted to a separate method or helper since you're using this in many tests